index.html

```diff
@@ -0,0 +1,74 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Brahim's Portfolio Page</title>
+    <!-- css links -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+</head>
+<body>
+   <header id="header">
+       <h1 class="headertitle">Jhon Doe</h1>
+<nav>
+    <img src="template5Images/Union.svg" alt="button">
+</nav>
+   </header>
+   <main class="main">
+  <section class="section1">
+  <div class="Names">
+      <h2 class="h2">Brahim Didi</h2>
+      <h3 class="h3">Software developer</h3>
+  </div>
+  <p class="paragraph">
+    I can help you build a product,
+    feature or website Look through
+    some of my work and experience! If
+    you like what you see and have a
+    project you need coded, don’t
+    hesitate to contact me.
+  </p>
+  <div class="skills-list">
+  <ul class="list">
+      <li class="skills"><h4>Languages</h4></li>
+      <li class="skills">Javascript</li>
+      <li class="skills">Ruby</li>
+      <li class="skills">Html</li>
+      <li class="skills">Css</li>
+  </ul>
+  <ul class="list">
+      <li class="skills"><h4>Frameworks</h4></li>
+      <li class="skills">Bootstrap</li>
+      <li class="skills">Ruby and rails</li>
+      <li class="skills">RSPec</li>
+      <li class="skills">CapaBara</li>
+      <li class="skills">Selenium</li>
+  </ul>
+  <ul class="list">
+      <li class="skills"><h4>Skills</h4></li>
+      <li class="skills">Codekit</li>
+      <li class="skills">Github</li>
+      <li class="skills">Codepen </li>
+      <li class="skills">Gitlab</li>
+      <li class="skills">Terminal</li>
+  </ul>
+  </div>
+  <div class="socials">
+      <ul class="sociallist">
```

Comment:
- [ ] So far for your project, I recommend you stick to the font and colors (background and font) defined for the template. Your page has a different background color and fonts.

Reply:
No that is completely wrong I took all the font and colors form the figma template, and they did not  say it should  be exactly the same.